docs/source/getting_started.rst

```diff
@@ -45,13 +51,15 @@ The **transformed training data** is usually just the prediction function applie
 
 The **log** is a dictionary, and can include any information that is relevant for inspecting or debugging the learner, e.g., what features were used, how many samples there were in the training set, feature importance or coefficients.
 
-Learner functions are usually partially initialized (curried) before being passed to pipelines or applied to data::
+Learner functions are usually partially initialized (curried) before being passed to pipelines or applied to data:
+
+.. code-block:: python
 
     from fklearn.training.regression import linear_regression_learner
     from fklearn.training.transformation import capper, floorer, prediction_ranger
 
     # initialize several learner functions
-    capper_fn = capper(columns_to_cap=["income"], precomputed_caps={"income": 50,000})
+    capper_fn = capper(columns_to_cap=["income"], precomputed_caps={"income": 50_000})
```

Comment:
Shouldn't this be?

```suggestion
    capper_fn = capper(columns_to_cap=["income"], precomputed_caps={"income": 50000})
```

Reply:
afaik [both are valid options](https://peps.python.org/pep-0515/), it is better with your suggestion, but I didn't feel confident enough to remove the separator.
If you think it is readable without the separator, it's ok for me

Reply:
I prefer without `_`, because that syntax is not usual.

Reply:
That's a good point, I will add your suggestion then @gcbeltramini